src/PlantsViewer.php

```diff
@@ -33,4 +21,16 @@ public static function displayAllPlants (array $plants): string
 
         return $output;
     }
+
+    public static function displaySinglePlant (Plant $plant): string
+    {
+        $output = "<div class='individual_plants'>";
+        $output .= "<h1>$plant->name</h1>";
+        $output .= "<p>$plant->family</p>";
+        $output .= "<img src='$plant->image' />";
+        $output .= "<p class='description'>$plant->description</p>";
+        $output .= "</div>";
+
+        return $output;
+    }
```

Comment:
Does this method ever actually get used? If not, you should remove it. Always good to keep on top of unused code